src/main/java/org/teamresistance/frc/Snorfle.java

```diff
@@ -0,0 +1,34 @@
+package org.teamresistance.frc;
+
+import org.strongback.command.Command;
+import org.strongback.components.Motor;
+
+/**
+ * @author Shreya Ravi
+ */
+public class Snorfle extends Command {
```

Comment:
On this note: I'd probably move all Snorfler-related stuff (commands included) to the `subsystem.snorfle` package, similar to how it's done in #4. I had tried the `commands` package approach but didn't like it, it makes scopes messy.